File: Structural/Lab11/C#/RealtorSubsystem.cs
using System;
using System.Collections.Generic;
using System.Text;
using C_.Entities;

namespace C_
{
    public class RealtorSubsystem
    {
        public string RealtorsPortfolio(List<Realtor> realtors)
        {
            var sb = new StringBuilder();
            foreach (var realtor in realtors)
            {
                sb.Append($"Realtor {realtor.NameAndSurname} has {realtor.RealEstates.Count} estates\n");
            }

            return sb.ToString();
        }
    }
}

File: Structural/Lab11/Typescript/src/realtor_subsystem.ts
import { Realtor } from "./Entities/realtor"

export class RealtorSubsystem {
    public realtorsPortfolio(realtors:Array<Realtor>):string{
        const sb: string[] = [];
        for (const realtor of realtors) {
            sb.push(`Realtor ${realtor.nameAndSurname} has ${realtor.RealEstates.length} estates\n`);
        }

        return sb.join("");
    }
}